cypress/e2e/person.js

```diff
@@ -4,7 +4,7 @@ describe('Person Visualization Check', () => {
         cy.location('pathname').should('eq', '/persons')
         cy.get('.ant-spin-spinning').should('not.exist') // Wait until initial table load to be able to use the search
         cy.get('[data-attr=persons-search]').type('deb').should('have.value', 'deb')
-        cy.get('.ant-input-search-button').click()
+        cy.wait(1000)
```

Comment:
we should look for something in the DOM. Which is normally quicker than always waiting for 1 sec